docs/reference/codelists.md

```diff
@@ -1,220 +1,212 @@
-   :header-rows: 1
-   :file: ../_static/patched/codelists/awardStatus.csv
 ```
 
-### bidStatus
+For more information on open and closed codelists, refer to the Open Contracting Data Standard [codelists documentation](http://standard.open-contracting.org/latest/en/schema/codelists/).
 
-```eval_rst
-.. csv-table-no-translate::
-   :header-rows: 1
-   :file: ../_static/patched/codelists/bidStatus.csv
-```
+## OCDS codelists
 
-### contractStatus
+OCDS for PPPs reuses some codelists from the Open Contracting Data Standard, without modification. Refer to the core OCDS documentation for details of these codelists:
 
-```eval_rst
-.. csv-table-no-translate::
-   :header-rows: 1
-   :file: ../_static/patched/codelists/contractStatus.csv
-```
+### Closed codelists
 
-### currency
+* [Award status](http://standard.open-contracting.org/latest/en/schema/codelists/#award-status)
+* [Contract status](http://standard.open-contracting.org/latest/en/schema/codelists/#contract-status)
+* [Currency](http://standard.open-contracting.org/latest/en/schema/codelists/#currency)
+* [Procurement method](http://standard.open-contracting.org/latest/en/schema/codelists/#method)
+* [Milestone status](http://standard.open-contracting.org/latest/en/schema/codelists/#milestone-status)
+* [Procurement category](http://standard.open-contracting.org/latest/en/schema/codelists/#procurement-category)
+* [Tender status](http://standard.open-contracting.org/latest/en/schema/codelists/#tender-status)
 
-```eval_rst
-.. csv-table-no-translate::
-   :header-rows: 1
-   :file: ../_static/patched/codelists/currency.csv
-```
+### Open Codelists
 
-### dataType
+* [Award criteria](http://standard.open-contracting.org/latest/en/schema/codelists/#award-criteria)
+* [Extended procurement category](http://standard.open-contracting.org/latest/en/schema/codelists/#extended-procurement-category)
+* [Item classification scheme](http://standard.open-contracting.org/latest/en/schema/codelists/#item-classification-scheme)
+* [Organization identifier scheme](http://standard.open-contracting.org/latest/en/schema/codelists/#organization-identifier-scheme)
+* [Related process](http://standard.open-contracting.org/latest/en/schema/codelists/#related-process)
+* [Related process scheme](http://standard.open-contracting.org/latest/en/schema/codelists/#related-process-scheme)
+* [Submission method](http://standard.open-contracting.org/latest/en/schema/codelists/#submission-method)
+* [Unit classification scheme](http://standard.open-contracting.org/latest/en/schema/codelists/#unit-classification-scheme)
 
-```eval_rst
-.. csv-table-no-translate::
-   :header-rows: 1
-   :file: ../_static/patched/codelists/dataType.csv
-```
+## OCDS for PPPs codelists
 
-### financeCategory
+OCDS for PPPs also modifies some codelists from OCDS core and uses codelists from individual extensions, including the OCDS for PPPs extension. The 'extension' column in the table display below indicates where each code originates from.
```

Comment:
> OCDS for PPPs modifies some codelists from OCDS and reuses codelists from extensions, without modification. The 'Extension' column in the tables below indicates where each code originates from.

I haven't checked whether the 'without modification' is true, but I added it to be consistent with our usage of 'reuses' in the previous section.

Reply:
I checked and it's true, updated in https://github.com/open-contracting-extensions/public-private-partnerships/pull/212/commits/d523f0f47929b7c4effe61ccce8d90e87110c5f5